src/main/java/com/attacktimer/AttackTimerBarOverlay.java

```diff
@@ -86,9 +91,23 @@ public Dimension render(Graphics2D graphics)
             ratio = (float)Math.max(1.0f - ratio, 0f);
         }
 
+        if (client.getSpriteOverrides().containsKey(SpriteID.HEALTHBAR_DEFAULT_FRONT_30PX)) {
+            final int barWidth = HD_FRONT_BAR.getWidth();
+            final int barHeight = HD_FRONT_BAR.getHeight();
+            final int barX = canvasPoint.getX() - barWidth / 2;
+            final int barY = canvasPoint.getY() + 7;
+
+            // Include padding so the bar doesn't show empty at very low prayer values
+            final int progressFill = (int) Math.ceil(Math.max(HD_Attack_BAR_PADDING * 2, Math.min((barWidth * ratio), barWidth)));
+
+            graphics.drawImage(HD_BACK_BAR, barX, barY, barWidth, barHeight, null);
+            // Use a sub-image to create the same effect the HD Health Bar has
+            graphics.drawImage(HD_FRONT_BAR.getSubimage(0, 0, progressFill, barHeight), barX, barY, progressFill, barHeight, null);
+            return null;
+        }
         // Draw bar
         final int barX = canvasPoint.getX() - 15;
-        final int barY = canvasPoint.getY();
+        final int barY = canvasPoint.getY() + 7;
```

Comment:
Why update this?

Reply:
So the bar doesn't overlap from a top view:
Before:
![image](https://github.com/user-attachments/assets/9badb69b-59d2-4b80-ad59-7abf3cf5e553)
After:
![image](https://github.com/user-attachments/assets/e5f437cd-dcb6-4ac3-ac66-0104432b45e0)